dts/msm8226/msm8926-v1-mtp.dts

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	qcom,msm-id = <200 0>;
+	qcom,board-id = <8 0>;
+
+	/* All the Nokia devices are expected to use lk1st, which passes the 
+	compatible to lk2nd, so no need for extra device matching. 
+	These nodes are listed in no particular order. */
+	nokia-tesla {
+		model = "Nokia Lumia 830";
+		compatible = "nokia,tesla", "qcom,msm8926", "lk2nd,device";
+
+		lk2nd,smd-rpm-hack-opening;
```

Comment:
I think this is not needed anymore on 5.18? Or cherry-pick https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=a8f8cc6b39b7ee0dbaccbebd1268c9d3458ebf13. I suppose it's fine for now but I will definitely drop this option in the new experimental-rebase branch.